### PR TITLE
CI: Add commitlint bot to travis CI

### PR DESCRIPTION
**Pull request by Madhu-1**
Add a commitlint to the Travis CI. to check the commit message format

Fixes: #1057

### CODE REVIEW

**Comment by Madhu-1** [comment]
@humblec can you please review this one, this helps to check the commit messages in CI

File: .travis.yml [hunk 1]
```diff
@@ -79,7 +79,11 @@ jobs:
         # install helm for helm lint
         - curl -L https://git.io/get_helm.sh
           | bash -s -- -v "${HELM_VERSION}"
+        - mkdir -p /opt/commitlint && pushd /opt/commitlint \
```
Comment (nixpanic) [inline .travis.yml:82]: missing `&&` between `pushd /opt/commitlint` and `npm init`, I do not think you need the `\` at the end of the line in yaml

**Comment by humblec** [comment]
@Madhu-1 yeah, apart from the change requested by @nixpanic , it looks good to me.

**Comment by Madhu-1** [comment]
@humblec PTAL

**Comment by humblec** [comment]
lgtm. thanks

**Pull request merged**